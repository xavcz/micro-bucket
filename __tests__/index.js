// Packages
const { test, expect } = global;
const micro = require('micro');
const listen = require('test-listen');
const fetch = require('node-fetch');

test('endpoint', async () => {
  const endpoint = micro(require('../src'));

  const url = await listen(endpoint);

  const rawResponse = await fetch(url);

  const response = await rawResponse.json();

  expect(response).toMatchSnapshot();
});
