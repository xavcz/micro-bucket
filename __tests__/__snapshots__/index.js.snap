// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`endpoint 1`] = `
Array [
  Object {
    "completed": true,
    "text": "Learn React Native 🚀",
  },
  Object {
    "completed": true,
    "text": "Load data asynchronously ⏲",
  },
  Object {
    "completed": true,
    "text": "Enjoy the React paradigm 🤙",
  },
  Object {
    "completed": false,
    "text": "Learn Redux tradeoffs ⚛",
  },
  Object {
    "completed": false,
    "text": "Make the app beautiful 🎨",
  },
  Object {
    "completed": false,
    "text": "Bridge native code with JavaScript 📲",
  },
  Object {
    "completed": false,
    "text": "Become a GraphQL hero ️️⚡",
  },
]
`;
